src/components/Question.js

```diff
@@ -15,9 +15,9 @@ const Question = ({ question, handleAnswerSelect, selectedAnswer }) => {
                 id={`${question.id}.${answer.id}`}
                 name={question.id}
                 value={answer.id}
-                onChange={handleAnswerSelect}
+                onChange={e => handleAnswerSelect(e, question.type)}
                 label={`${answer.text}`}
-                checked={answer.id === selectedAnswer}
+                checked={selectedAnswer && selectedAnswer.includes(answer.id)}
```

Comment:
This line of code is a reason for re warning in the console. On first render this `selectedAnswer && selectedAnswer.includes(answer.id)` returns undefined. To fix this you need to change this line to the following
```
checked={
                  selectedAnswer ? selectedAnswer.includes(answer.id) : false
                }
```

Reply:
ah, I see, thank you @kosenchiha! I'll updated it accordingly

Reply:
I think the root fix for this is to use `defaultChecked` instead of `checked` to turn it from a controlled to uncontrolled component. See the React docs on uncontrolled components: https://reactjs.org/docs/uncontrolled-components.html#default-values

I remember you initially made this attribute `defaultChecked` @kosenchiha but since I wasn't aware of the information above I suggested you to change it to `checked`. My mistake, sorry!